src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1054,6 +1056,9 @@ export class StudyViewPageStore {
                 case ChartTypeEnum.MUTATED_GENES_TABLE:
                     this.resetGeneFilter();
                     break;
+                case ChartTypeEnum.FUSION_GENES_TABLE:
+                    this.resetGeneFilter();
```

Comment:
just confirming that we don't have separate FusionGene filter?

Reply:
we do not. Fusions are not properly tracked in the database at the moment. This is really a temporary solution to address GENIE study mutations over counting problem.

Reply:
update: discussed with @zhx828, added a new gene filter for fusions.